### PR TITLE
one new rule added and change to an existing rule

### PR DESCRIPTION
**Pull request by aprakash13**
Fixes #

## Proposed Changes

  -
  -
  -

### CODE REVIEW

File: Detections/OfficeActivity/Malicious_Inbox_Rule.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+   This is done so as to limit ability to warn compromised users that they've been compromised. Below is a sample query that tries to detect this.
+  Reference: https://www.reddit.com/r/sysadmin/comments/7kyp0a/recent_phishing_attempts_my_experience_and_what/'
+requiredDataConnectors:
+  - connectorId: Office365
+    dataTypes:
+      - OfficeActivity
+tactics:
+  - Defense Evasion
+relevantTechniques:
+  -
```
Comment (shainw) [inline Detections/OfficeActivity/Malicious_Inbox_Rule.yaml]: Technique?

File: Detections/OfficeActivity/Malicious_Inbox_Rule.yaml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  - connectorId: Office365
+    dataTypes:
+      - OfficeActivity
+tactics:
+  - Defense Evasion
+relevantTechniques:
+  -  
+query: |
+
+ let timeframe = 14d;
```
Comment (shainw) [inline Detections/OfficeActivity/Malicious_Inbox_Rule.yaml]: I don't think we need a detection for 14d, this should be 1d

File: Detections/OfficeActivity/Malicious_Inbox_Rule.yaml [hunk 3]
```diff
@@ -0,0 +1,34 @@
+id: 7b907bf7-77d4-41d0-a208-5643ff75bf9a
+name: Malicious Inbox Rule
+description: |
+  'Often times after the initial compromise the attackers create inbox rules to delete emails that contain certain keywords. 
+   This is done so as to limit ability to warn compromised users that they've been compromised. Below is a sample query that tries to detect this.
+  Reference: https://www.reddit.com/r/sysadmin/comments/7kyp0a/recent_phishing_attempts_my_experience_and_what/'
```
Comment (shainw) [inline Detections/OfficeActivity/Malicious_Inbox_Rule.yaml]: Missing standard values for detections -  Severity, queryFrequency, queryPeriod, triggerOperator, triggerThreshold

File: Hunting Queries/OfficeActivity/OfficeMailForwarding_hunting.yaml [hunk 4]
```diff
@@ -1,5 +1,5 @@
 id: 871ba14c-88ef-48aa-ad38-810f26760ca3
-name: Long lookback for multiple users forwarding Office mail to same destination
+name: UOffice Mail Forwarding
```
Comment (shainw) [inline Hunting Queries/OfficeActivity/OfficeMailForwarding_hunting.yaml]: Looks like you accidentally put a "U" at the beginning of the name

File: Hunting Queries/OfficeActivity/OfficeMailForwarding_hunting.yaml [hunk 5]
```diff
@@ -1,5 +1,5 @@
...
 description: |
   'This query over Office Activity audit data highlights cases where user mail is being forwarded.'
```
Comment (shainw) [inline Hunting Queries/OfficeActivity/OfficeMailForwarding_hunting.yaml]: Can you update the description a bit also.  Probably needs some mention of why this is interesting.

File: Hunting Queries/OfficeActivity/OfficeMailForwarding_hunting.yaml [hunk 6]
```diff
@@ -18,9 +18,21 @@ query: |
   OfficeActivity
   | where TimeGenerated >= ago(timeframe)
   | where (Operation == "Set-Mailbox" and Parameters contains 'ForwardingSmtpAddress') 
-      or (Operation == 'New-InboxRule' and Parameters contains 'ForwardTo')
+  or (Operation == 'New-InboxRule' and Parameters contains 'ForwardTo')
```
Comment (shainw) [inline Hunting Queries/OfficeActivity/OfficeMailForwarding_hunting.yaml]: =~

File: Hunting Queries/OfficeActivity/OfficeMailForwarding_hunting.yaml [hunk 7]
```diff
@@ -18,9 +18,21 @@ query: |
...
   | extend parsed=parse_json(Parameters)
-  | extend fwdingDestination = iif(Operation=="Set-Mailbox", tostring(parsed[1].Value), tostring(parsed[2].Value))
-  | where fwdingDestination != ""
-  | project TimeGenerated, UserId, Operation, fwdingDestination, ClientIP 
-  | extend timestamp = TimeGenerated, AccountCustomEntity = UserId, IPCustomEntity = ClientIP
+  | extend fwdingDestination_initial = (iif(Operation=="Set-Mailbox", tostring(parsed[1].Value), tostring(parsed[2].Value)))
```
Comment (shainw) [inline Hunting Queries/OfficeActivity/OfficeMailForwarding_hunting.yaml]: =~

File: Hunting Queries/OfficeActivity/OfficeMailForwarding_hunting.yaml [hunk 8]
```diff
@@ -18,9 +18,21 @@ query: |
   OfficeActivity
   | where TimeGenerated >= ago(timeframe)
   | where (Operation == "Set-Mailbox" and Parameters contains 'ForwardingSmtpAddress')
```
Comment (shainw) [inline Hunting Queries/OfficeActivity/OfficeMailForwarding_hunting.yaml]: =~

File: Hunting Queries/OfficeActivity/OfficeMailForwarding_hunting.yaml [hunk 9]
```diff
@@ -18,9 +18,21 @@ query: |
...
-  | where fwdingDestination != ""
-  | project TimeGenerated, UserId, Operation, fwdingDestination, ClientIP 
-  | extend timestamp = TimeGenerated, AccountCustomEntity = UserId, IPCustomEntity = ClientIP
+  | extend fwdingDestination_initial = (iif(Operation=="Set-Mailbox", tostring(parsed[1].Value), tostring(parsed[2].Value)))
+  | where isnotempty(fwdingDestination_initial)
+  | extend fwdingDestination = iff(fwdingDestination_initial has "smtp", (split(fwdingDestination_initial,":")[1]), fwdingDestination_initial )
+  | parse fwdingDestination with * '@' ForwardedtoDomain 
+  | parse UserId with *'@' UserDomain
+  | extend subDomain = ((split(strcat(tostring(split(UserDomain, '.')[-2]),'.',tostring(split(UserDomain, '.')[-1])), '.') [0]))
+  | where ForwardedtoDomain !contains  subDomain
```
Comment (shainw) [inline Hunting Queries/OfficeActivity/OfficeMailForwarding_hunting.yaml]: extra space after !contains

File: Hunting Queries/OfficeActivity/OfficeMailForwarding_hunting.yaml [hunk 10]
```diff
@@ -18,9 +18,21 @@ query: |
...
+  | extend Result = iff( ForwardedtoDomain != UserDomain ,"Mailbox rule created to forward to External Domain", "Forward rule for Internal domain")
+  | extend ClientIPAddress = case( ClientIP has ".", tostring(split(ClientIP,":")[0]), ClientIP has "[", tostring(trim_start(@'[[]',tostring(split(ClientIP,"]")[0]))), ClientIP )
+  | extend Port = case(
+  ClientIP has ".", (split(ClientIP,":")[1]),
+  ClientIP has "[", tostring(split(ClientIP,"]:")[1]),
+  ClientIP
+  )
+  | project TimeGenerated, UserId, UserDomain, subDomain, Operation, ForwardedtoDomain, ClientIPAddress ,  Result, Port, OriginatingServer, OfficeObjectId, fwdingDestination
```
Comment (shainw) [inline Hunting Queries/OfficeActivity/OfficeMailForwarding_hunting.yaml]: Fix up extra spacing in project

Comment (aprakash13) [reply Hunting Queries/OfficeActivity/OfficeMailForwarding_hunting.yaml]: Done. Thanks Shain.

File: Hunting Queries/OfficeActivity/OfficeMailForwarding_hunting.yaml (hunk 9 above)
Comment (aprakash13) [reply Hunting Queries/OfficeActivity/OfficeMailForwarding_hunting.yaml]: Done. Thanks Shain.

File: Hunting Queries/OfficeActivity/OfficeMailForwarding_hunting.yaml (hunk 8 above)
Comment (aprakash13) [reply Hunting Queries/OfficeActivity/OfficeMailForwarding_hunting.yaml]: Done. Thanks Shain.

File: Hunting Queries/OfficeActivity/OfficeMailForwarding_hunting.yaml (hunk 7 above)
Comment (aprakash13) [reply Hunting Queries/OfficeActivity/OfficeMailForwarding_hunting.yaml]: Done. Thanks Shain.

File: Hunting Queries/OfficeActivity/OfficeMailForwarding_hunting.yaml (hunk 6 above)
Comment (aprakash13) [reply Hunting Queries/OfficeActivity/OfficeMailForwarding_hunting.yaml]: Done. Thanks Shain.

File: Hunting Queries/OfficeActivity/OfficeMailForwarding_hunting.yaml (hunk 5 above)
Comment (aprakash13) [reply Hunting Queries/OfficeActivity/OfficeMailForwarding_hunting.yaml]: Done Now. Thanks.

File: Hunting Queries/OfficeActivity/OfficeMailForwarding_hunting.yaml (hunk 4 above)
Comment (aprakash13) [reply Hunting Queries/OfficeActivity/OfficeMailForwarding_hunting.yaml]: Fixed. Thanks.

File: Detections/OfficeActivity/Malicious_Inbox_Rule.yaml (hunk 3 above)
Comment (aprakash13) [reply Detections/OfficeActivity/Malicious_Inbox_Rule.yaml]: Added the suggested details. Thanks Shain.

File: Detections/OfficeActivity/Malicious_Inbox_Rule.yaml (hunk 2 above)
Comment (aprakash13) [reply Detections/OfficeActivity/Malicious_Inbox_Rule.yaml]: Done now.

Comment (aprakash13) [reply Detections/OfficeActivity/Malicious_Inbox_Rule.yaml]: Thanks

File: Detections/OfficeActivity/Malicious_Inbox_Rule.yaml (hunk 1 above)
Comment (aprakash13) [reply Detections/OfficeActivity/Malicious_Inbox_Rule.yaml]: Wasn't sure if there  might be a technique under defense evasion that would be an exact match.  The closest I could find was : https://attack.mitre.org/techniques/T1089/  as it had the word event reporting. However, this also didn't seem like a good match. Hence left it blank

**Pull request merged**